logging/README.md

```diff
@@ -23,15 +23,20 @@ These components are deployed:
 Provides detailed Elasticsearch performance information for Prometheus
 
 ## Perform Pre-Deployment Tasks
-Before deploying, you must select the release that you want to deploy, then create a local copy of the repository. 
 
-### Select the Release to Copy
+Before deploying, you must perform these tasks:
+
+- [select the release that you want to deploy](#log_sel_rel)
+- [create a local copy of the repository](#log_loc_copy)
+- [customize your deployment](#log_custom)
+
+### <a name="log_sel_rel"></a>Select the Release to Copy
 
 1. Click on **tags** above the repository tree.
 2. On the **Tags** page, click [Releases](https://github.com/sassoftware/viya4-monitoring-kubernetes/releases) to view the list of available releases.
 3. Use the release notes to determine the release you want to deploy.
```

Comment:
Should we mention the 'stable' release here? It's not under 'Tags' but I think it's the recommended version users should download. I guess if they're going to 'Tags' they'll see the latest stable release anyway...

Reply:
Added a step to select the "stable" release.

Reply:
Does changing the branch affect which releases are available?  I'm not sure it does.  In any cases, I think we want people to click on the Releases link on the right hand side of the initial repository screen view (directly under the About text). This brings up the Release notes for each release _without_ requiring them to then click on Releases (i.e. saves a step).